contracts/cw1-whitelist/src/contract.rs

```diff
@@ -62,6 +68,29 @@ impl Cw1WhitelistContract {
 
         Ok(Response::new())
     }
+
+    #[msg(exec)]
+    fn remove_member(
+        &self,
+        (deps, _env, msg): (DepsMut, Env, MessageInfo),
+        member: String,
+    ) -> Result<Response, ContractError> {
+        self.members
+            .remove(deps.storage, deps.api.addr_validate(&member)?);
```

Comment:
Using &Addr as Map key could be more efficient.
For example, if we used msg.sender here (which is also used below in the events emitted), we would need:

`self.members.remove(deps.storage, msg.sender.clone());`

With reference, only:

`self.members.remove(deps.storage, &msg.sender);`

And they both accomplish the same work under the hood. One just forces unnecessary clone